[PR-17382] Add sharer_info to Post-Share Webhook (#261)
talkable/talkable-docs: source/web_hooks/post_share.rst

    * **recipients** — an array of email addresses

> * **sharer_info** — subhash of parameters describing the person sending the share

Reviewer: ```suggestion
* **sharer_info** — subhash of parameters describing the person who shared the offer
```

  .. include:: /partials/person_fields.rst

* **share_link** — link sent in share message (optional, present only for
  Facebook and Twitter shares)
* **origin** — contains data about the event that issued an offer:

  .. include:: /partials/origin_fields.rst

> Note: origin email is the email at the time when the campaign is first shown to the visitor, please, use sharer info for the most up to date information

Reviewer: * Use `..note:` directive.
* We can elaborate on this, please approve the text with Stacey if you proceed with this suggestion.
```suggestion
.. note::

   ``origin.email`` contains the email saved at the moment when the campaign is 
   first shown to the Advocate. It could be ``null``. For the most up-to-date 
   information about the Advocate, use ``sharer_info`` property which is updated
   with the email address the Advocate has entered on the Advocate Signup Page.

Author: The new text was approved with Stacey

     "share_info": {
       "recipients": ["john@example.com"]
     },
     "sharer_info": {
       "first_name": "Bob",
       "last_name": "Smith",
       "email": "sharer@example.com",
       "username": "username",
       "unsubscribed_at": null,
       "subscribed_at": "2018-09-27T22:55:13.860+03:00",
       "opted_in_at": "2018-09-27T22:55:13.860+03:00",
       "sub_choice": true,
       "referral_counts": {
         "total": 0,
         "approved": 0,
         "pending": 0
       },
       "is_loyalty_member": false,
       "loyalty_member": null
     },
     "origin": {
       "id": 147886587,
       "type": "Purchase",
       "order_number": "450901776",
       "subtotal": 35.03,
       "email": "sharer@example.com",
       "customer_id": "565659001",

>        "order_date":"2019-04-11T07:28:31.258-07:00",

Reviewer: ```suggestion
       "order_date": "2019-04-11T07:28:31.258-07:00",
```

     "share_info": {
       "recipients": ["john@example.com"]
     },
     "sharer_info": {
       "first_name": "Bob",
       "last_name": "Smith",
       "email": "sharer@example.com",
       "username": "username",
       "unsubscribed_at": null,
       "subscribed_at": "2018-09-27T22:55:13.860+03:00",
       "opted_in_at": "2018-09-27T22:55:13.860+03:00",
       "sub_choice": true,
       "referral_counts": {
         "total": 0,
         "approved": 0,
         "pending": 0
       },
       "is_loyalty_member": false,
       "loyalty_member": null
     }
     "origin": {

>        "id": 147886587,

Reviewer: @hmlON There’s no `origin.email` here, please fix the example.
